eclipse-collections/src/main/java/org/eclipse/collections/impl/utility/internal/SetIterables.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2021 Goldman Sachs and others.
+ * Copyright (c) 2022 Goldman Sachs and others.
```

Comment:
I think you meant to replace UnifiedSet with Sets in here.

Reply:
I reverted the change and forgot to revert the copyright. I'll take care of this shortly.